src/script/engine.zig

```diff
@@ -264,6 +266,22 @@ pub const Engine = struct {
         return error.EarlyReturn;
     }
 
+    /// OP_TOALTSTACK: Puts the value onto the top of the alt stack, and removes it from the main stack.
+    ///
+    fn opToAltStack(self: *Engine) !void {
+        const value = try self.stack.pop();
+        defer self.allocator.free(value);
+        try self.alt_stack.pushByteArray(value);
+    }
+    
+    /// OP_FROMALTSTACK: Puts the value onto the top of the main stack, and removes it from the alt stack.
+    ///
+    fn opFromAltStack(self: *Engine) !void {
+        const value = try self.alt_stack.pop();
+        defer self.allocator.free(value);
+        try self.stack.pushByteArray(value);
```

Comment:
Please read this discussion: https://github.com/zig-bitcoin/btczee/pull/34#discussion_r1761600214

Once the PR it belongs to will be merged, you will be able to use the `pushElement` method directly.

I will ping you when it's ready

Reply:
I've been looking into this very carefully but I'm going to pave the way, and then update when #47 is merged.